osu.Game/Overlays/Chat/Tabs/ChannelSelectorTabItem.cs

```diff
@@ -39,6 +39,7 @@ public class ChannelSelectorTabChannel : Channel
             public ChannelSelectorTabChannel()
             {
                 Name = "+";
+                Type = ChannelType.Temporary;
```

Comment:
Is there a reason why you're using the temporary type rather than a new "system" type or something like that?

Reply:
No reason honestly, it just didn't cross my mind at all. Will do.